app/components/title-input.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['input-edit'],
+  classNameBindings: ['active'],
+
+  //proxy clicks to input focus
+  click() {
+    this.$('input').focus();
+  },
+
+  actions: {
+    inputFocusIn() {
+      this.set('active', true);
+    },
+
+    inputFocusOut() {
+      this.set('active', false);
+    }
+  }
```

Comment:
@Gaurav0 @joostdevries 

I tried to.
We want to style a parent element depending on a **child** state. I don't think we can we do that with css. Can we?

The focus "proxying" part I'm pretty sure that can't be done with css. Remember we know have a container that holds both the input and the pencil icon.